hack/addons/mindthegap-helm-registry/Dockerfile

```diff
@@ -3,10 +3,14 @@ FROM --platform=${BUILDPLATFORM} ghcr.io/mesosphere/mindthegap:${MINDTHEGAP_VERS
 # this gets called by goreleaser so the copy source has to be the path relative to the repo root.
 RUN --mount=source=./hack/addons/mindthegap-helm-registry/repos.yaml,target=/repos.yaml \
     ["/ko-app/mindthegap", "create", "bundle", "--helm-charts-file=/repos.yaml", "--output-file=/tmp/helm-charts.tar"]
+# TODO remove me as soon as its not needed to hold multiple versions of helm charts
+FROM ghcr.io/nutanix-cloud-native/caren-helm-reg:v0.14.7 as carenprevious
```

Comment:
I think we want [v0.14.6](https://github.com/mesosphere/konvoy2/tree/v2.12.0/pkg/capi/client/builtinrepo/components/runtime-extension-caren/v0.14.6) and [v0.14.9](https://github.com/mesosphere/konvoy2/tree/v2.12.1/pkg/capi/client/builtinrepo/components/runtime-extension-caren/v0.14.9)?